costmap_cspace/src/costmap_3d.cpp

```diff
@@ -102,21 +102,16 @@ class Costmap3DOFNode
     map->processMapOverlay(msg);
     ROS_DEBUG("C-Space costmap updated");
   }
-  bool cbUpdateStatic(
-      const costmap_cspace::CSpace3DMsg::Ptr map,
-      const costmap_cspace_msgs::CSpace3DUpdate::Ptr update)
+  bool cbUpdateStatic(const costmap_cspace::CSpace3DMsg::Ptr map)
   {
     publishDebug(*map);
     pub_costmap_.publish<costmap_cspace_msgs::CSpace3D>(*map);
     return true;
   }
-  bool cbUpdate(
-      const costmap_cspace::CSpace3DMsg::Ptr map,
-      const costmap_cspace_msgs::CSpace3DUpdate::Ptr update)
+  bool cbUpdate(const costmap_cspace_msgs::CSpace3DUpdate::Ptr update)
   {
     if (update)
     {
-      publishDebug(*map);
```

Comment:
I think updated map should be reflected to the debug output instead of the static map

Reply:
Done. Please take another look.